openreview/openreview-web
# Initial test to run unit tests with an empty database

How to run the tests:

from openreview repo, until I merge the draft PR https://github.com/openreview/openreview/pull/2135, you need to checkout the branch `fix/param-urls` and change the configuration of circleci.json to use the new urls for the registration:

```
NODE_ENV=circleci node scripts/clean_start_app.js
```

from openreview-web:

```
testcafe chrome -sf tests/regisrer.ts
```

circleci.json changes:

```
    "activateProfilePath": "/profile/activate",
    "activateAlternateEmailPath": "/confirm",
    "resetPasswordPath": "/account/password"
    "frontendUrl": "http://localhost:3030",
```
